visallo/visallo
# Add to the sample ontology

I found that when I was trying to create a sample RDF file to ingest, these new properties, concept, and relationships allowed me to create a slightly more realistic scenario.

- [x] mwizeman joeybrk372 rygim jharwig EvanOxfeld

Testing Instructions:
1.) Follow the instructions in the [Getting Started](http://docs.visallo.org/getting-started.html) documentation to start a Tomcat server.
2.) Log in to Visallo and verify that there is a new Person concept type.
3.) Create a new Person, Zipcode, and Email Address on a graph and link them with "Has Contact Info" and "Lives in" edges.

CHANGELOG
Changed: Added a "Person" concept as well as "Has Contact Info" and "Lives in" relationship types to the sample ontology.
